src/lib/client.js

```diff
@@ -98,10 +101,12 @@ class Client extends EventEmitter {
       }
 
       const quoteQuery = {
-        source_ledger: plugin.id,
+        source_address: (yield plugin.getPrefix()),
```

Comment:
The core client just passes through options to the `plugin` constructor. `prefix` isn't actually required for `Client` or the Ledger plugin in general, it just happens to be required for ilp-plugin-bells.

Since `getPrefix()` as described in https://github.com/interledger/rfcs/issues/31 returns a `Promise` I assumed that `prefix` is not necessarily a parameter for all ledger plugins.